pkg/mount-manager/safe-mounter_windows.go

```diff
@@ -251,3 +251,11 @@ func (mounter *CSIProxyMounter) ExistsPath(path string) (bool, error) {
 		})
 	return isExistsResponse.Exists, err
 }
+
+func (mounter *CSIProxyMounter) GetBlockSizeBytes(diskId string) (int64, error) {
+	DiskStatsResponse, err := mounter.DiskClient.DiskStats(context.Background(),
```

Comment:
You mentioned this will return a smaller size than the actual disk? Will that cause problems with resize returning success?

Reply:
Before I was using volumestats API, now I changed to DiskStats which will return the full disk size. So the number can match now.